lib/parse_date/int_from_string.rb

```diff
@@ -76,14 +81,14 @@ def self.year_int_valid?(year)
     BRACKETS_BETWEEN_DIGITS_REXEXP = Regexp.new('\d[' + Regexp.escape('[]') + ']\d')
 
     # removes brackets between digits such as 169[5] or [18]91
-    def remove_brackets(orig_date_str)
-      orig_date_str.delete('[]') if orig_date_str.match(BRACKETS_BETWEEN_DIGITS_REXEXP)
+    def remove_brackets(date_str)
+      date_str.delete('[]') if date_str.match(BRACKETS_BETWEEN_DIGITS_REXEXP)
```

Comment:
Mind changing `s/REXEXP/REGEXP/` on this line and on line 81?

Reply:
hahah - i didn't even see it. thanks!